Makefile

```diff
@@ -134,6 +135,8 @@ publish_contacts_admin:
 
 publish_whitehall:
 	$(DOCKER_COMPOSE_CMD) exec -T whitehall-admin bundle exec rake publishing_api:publish_special_routes
+populate_end_to_end_test_data_from_whitehall:
```

Comment:
Maybe add a blank line above this one, to improve readability?

Reply:
@cbaines Updated and fixed-up.